core/arch/x86/opcode.h

```diff
@@ -1259,10 +1259,14 @@ enum {
 /* 1101 */     OP_vpbroadcastd,   /**< IA-32/AMD64 vpbroadcastd opcode. */
 /* 1102 */     OP_vpbroadcastq,   /**< IA-32/AMD64 vpbroadcastq opcode. */
 
+    /* Added in Skylake */
```

Comment:
Best to match the comment in decode_table.c -- for bigger changes I end up using x86opnums.pl and pasting from one to the other which will clobber any difference at that point.